tests/unit/ConfigurationActions.spec.ts

```diff
@@ -7,7 +7,9 @@ import { nextTick } from 'vue'
 // mock the console.info function to hide the output
 console.info = jest.fn()
 
-describe('Fetch Config', () => {
+describe('Configuration Actions', () => {
+  const store = getVuexStore() as any // remove typings for unit tests
```

Comment:
Should be done just once per test suite, or you get duplicate stores.

Reply:
Good point!